packages/trace-viewer/index.html

```diff
@@ -26,5 +26,12 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/index.tsx"></script>
+    <dialog id="fallback-error">
+      <p>The Playwright Trace Viewer must be loaded over the <code>http://</code> or <code>https://</code> protocols.</p>
+    </dialog>
+    <script>
+      if (!/^https?:/.test(window.location.protocol))
+        document.getElementById("fallback-error").show();
+    </script>
```

Comment:
We can probably be more helpful—although there's a few ways to get here. Realistically, telling people to run `npx playwright show-report` would probably be the most helpful or running/hosting through static server.

Reply:
yes adding documentation in here as a message would be fantastic. Telling them why and how to solve it so  and linking to the docs page would be great